include/packetgraph/syscall_catcher.h

```diff
@@ -0,0 +1,31 @@
+/* Copyright 2019 Outscale SAS
+ *
+ * This file is part of Packetgraph.
+ *
+ * Packetgraph is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as published
+ * by the Free Software Foundation.
+ *
+ * Packetgraph is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Packetgraph.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef _PG_SYSCALL_CATCHER_H
+#define _PG_SYSCALL_CATCHER_H
+
+#include <sys/prctl.h>
+#include <linux/seccomp.h>
+
+/**
+ * Initialize syscall filter
+ *
+ * @return 0 if the filter has been correctly build, 1 on the contrary.
+ */
+int pg_init_syscall_catcher(void);
```

Comment:
I think the rule is 0 on success and -1 on error 
also you can use a pg_error to set error